e2e/smoke/MODULE.bazel

```diff
@@ -1,22 +1,26 @@
 bazel_dep(name = "rules_distroless", version = "0.0.0", dev_dependency = True)
-bazel_dep(name = "bazel_skylib", version = "1.3.0", dev_dependency = True)
+bazel_dep(name = "bazel_skylib", version = "1.5.0", dev_dependency = True)
 bazel_dep(name = "platforms", version = "0.0.10", dev_dependency = True)
-
-bazel_dep(name = "rules_oci", version = "1.7.4")
-bazel_dep(name = "container_structure_test", version = "1.16.0")
-bazel_dep(name = "aspect_bazel_lib", version = "2.7.3")
+bazel_dep(name = "rules_oci", version = "2.0.0-beta1", dev_dependency = True)
+bazel_dep(name = "container_structure_test", version = "1.16.0", dev_dependency = True)
+bazel_dep(name = "aspect_bazel_lib", version = "2.7.3", dev_dependency = True)
 
 local_path_override(
     module_name = "rules_distroless",
     path = "../..",
 )
 
 apt = use_extension("@rules_distroless//apt:extensions.bzl", "apt")
-apt.deb_index(
+apt.install(
     name = "bullseye",
     lock = ":bullseye.lock.json",
     manifest = ":bullseye.yaml",
 )
+apt.install(
+    name = "bullseye_nolock",
```

Comment:
nit: maybe comment about why this is needed?

Reply:
Sure, the doc here explains why. i will leave comment there for visibility. https://github.com/GoogleContainerTools/rules_distroless/blob/main/docs/apt.md#lockfile-generation